client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -333,7 +333,7 @@ export const getCustomsErrors = (
 			} );
 
 			if ( pckg.itn ) {
-				if ( ! /^(AES X\d{14})|(NOEEI 30\.\d{1,2}(\([a-z]\)(\(\d\))?)?)$/.test( pckg.itn ) ) {
+				if ( ! /^((AES X\d{14})|(NOEEI 30\.\d{1,2}(\([a-z]\)(\(\d\))?)?))$/.test( pckg.itn ) ) {
```

Comment:
Unrelated, but I discovered it while testing another PR. You can check in https://regex101.com/, before you could write anything after a valid ITN (for example, `AES X12345678901234aaaaaaaaaaa!`), now it validates correctly.

Reply:
Since we're making changes to the regex - should we also change the capturing groups to be non-capturing? `/^(?:(?:AES X\d{14})|(?:NOEEI 30\.\d{1,2}(?:\([a-z]\)(?:\(\d\))?)?))$/` ?